src/main/java/com/launchacademy/reviews/services/VenueService.java

```diff
@@ -0,0 +1,26 @@
+package com.launchacademy.reviews.services;
+
+import com.launchacademy.reviews.models.Venue;
+import com.launchacademy.reviews.repositories.VenueRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class VenueService {
+private VenueRepository venueRepository;
+
+  @Autowired
+  public VenueService( VenueRepository venueRepository) {
+    this.venueRepository = venueRepository;
+  }
+
+public List<Venue> findAllVenues(){
```

Comment:
please fix indentation here:
- Line 12
- Lines 19-25

And remove extra empty whitespace from line 15 (before `VenueRepository`)!